src/app/space/add-space-overlay/add-space-overlay.component.ts

```diff
@@ -60,7 +60,9 @@ export class AddSpaceOverlayComponent implements OnInit {
     }));
     this.subscriptions.push(this.spaceTemplateService.getSpaceTemplates()
       .subscribe((templates: ProcessTemplate[]) => {
-        this.spaceTemplates = templates.filter(t => t.attributes['can-construct']);
+        this.spaceTemplates = templates.filter(t => {
+          return t.attributes['can-construct'] && (t.attributes.name === 'Agile' || t.id === 'f405fa41-a8bb-46db-8800-2dbe13da1418');
```

Comment:
This ID feels like a risk. What happens if the ID changes on the backend.

Reply:
I can accept that this is the primary key and shouldn't change, it still just feels odd having a PK in the UI.

Reply:
that should really be abstracted away

Reply:
Do you have any other suggestions on how we could filter out just `Agile` template?

Reply:
no, maybe @kwk can answer that

Reply:
If you don't send a template at all we will default for SDD on the backend: https://github.com/fabric8-services/fabric8-wit/blob/8674b7b56a9226b3bbd6ebf7b0179f127ff6a0a2/controller/space.go#L96 . Maybe it is time to change this to the Agile template and then you don't have to send anything?

Reply:
@kwk That would be a better solution in my opinion. What do you think @joshuawilson?

Reply:
yes, changing the backend to default to Agile is much cleaner

Reply:
@joshuawilson I've create a task here https://github.com/fabric8-services/fabric8-wit/issues/2317